docs/_writing_dependencies_conda.rst

```diff
@@ -0,0 +1,318 @@
+    2017-02-22 10:13:29,516 DEBUG [galaxy.tools.deps] Using dependency samtools version None of type conda
+    2017-02-22 10:13:29,516 DEBUG [galaxy.tools.deps] Using dependency samtools version None of type conda
+    ok
+    
+    ----------------------------------------------------------------------
+    XML: /private/tmp/tmpLvKwta/xunit.xml
+    ----------------------------------------------------------------------
+    Ran 1 test in 15.936s
+    
+    OK
+    2017-02-22 10:13:37,014 INFO  [test_driver] Shutting down
+    2017-02-22 10:13:37,014 INFO  [test_driver] Shutting down embedded galaxy web server
+    2017-02-22 10:13:37,016 INFO  [test_driver] Embedded web server galaxy stopped
+    2017-02-22 10:13:37,017 INFO  [test_driver] Stopping application galaxy
+    ....
+    2017-02-22 10:13:37,018 INFO  [galaxy.jobs.handler] sending stop signal to worker thread
+    2017-02-22 10:13:37,018 INFO  [galaxy.jobs.handler] job handler stop queue stopped
+    Testing complete. HTML report is in "/Users/john/workspace/planemo/project_templates/seqtk_complete/tool_test_output.html".
+    All 1 test(s) executed passed.
+    seqtk_seq[0]: passed
+
+In this case the tests passed and the line containing ``[galaxy.tools.deps] Using dependency seqtk version 1.2 of type conda``
+indicates Galaxy dependency resolution was successful and it found the environment we previously installed with ``conda_install``.
+
+----------------------------------------------------------------
+Finding Existing Conda Packages
+----------------------------------------------------------------
+
+How did we know what software name and software version to use? We found the existing
+packages available for Conda and referenced them. To do this yourself, you can simply
+use the planemo command ``conda_search``. If we do a search for ``seqt`` it will show
+all the software and all the versions available matching that search term - including
+``seqtk``.
+
+::
+
+    $ planemo conda_search seqt
+    Fetching package metadata ...............
+    seqtk                        r75                           0  bioconda
+                                 r82                           0  bioconda
+                                 r93                           0  bioconda
+                                 1.2                           0  bioconda
+
+.. note:: The Planemo command ``conda_search`` is a light wrapper around the underlying
+   ``conda search`` command but configured to use the same channels and other options as
+   Planemo and Galaxy. The following Conda command would also work to search::
+
+       $ $HOME/miniconda3/bin/conda -c bioconda -c conda-forge -c iuc seqt
```

Comment:
Highest priority is iuc, then bioconda, then conda-forge. See https://github.com/galaxyproject/galaxy/blob/dev/config/galaxy.ini.sample#L212.
So that would be `-c iuc -c bioconda -c conda-forge` (I'm excluding defaults here, assuming this is always active). Can you confirm this @bgruening ?